javaScriptFunctions_arrowFunctions/codeChallenge/random.js

```diff
@@ -0,0 +1,11 @@
+function generateRandomNumber( high, low ) {
```

Comment:
@nicoRDL, you can also remove the spaces after the opening parenthesis and before the closing parenthesis:
`function generateRandomNumber(high, low)`

Reply:
PR updated for review @lycoangelo 

https://github.com/nicoRDL/dotTheory-JS/pull/10/commits/ee1d5a4e3fecc6458bc21b11ce52ee189b8cc7de